### PR TITLE
feat: mfe multitenant config

### PR DESCRIPTION
**Pull request by dcoa**
**Description:**

This PR adds a second query param to identify the tenant and the API could send their specific configurations.

This implementation is based on version 1 explained [here](https://app.tryeraser.com/integration/gather/tRYtDCbPfImR6xD2table0) .

### Note
This PR is only for review coding and can't be marge. 

Jirs card DS-129: https://edunext.atlassian.net/browse/DS-129

### CODE REVIEW

File: src/initialize.test.js [hunk 1]
```diff
@@ -279,14 +293,16 @@ describe('initialize', () => {
...
+  it('should initialize the app with runtime configuration by site', async () => {
     config.MFE_CONFIG_API_URL = 'http://localhost:18000/api/mfe/v1/config';
     config.APP_ID = 'auth';
     configureCache.mockReturnValueOnce(Promise.resolve({
       get: (url) => {
         const params = new URL(url).search;
         const mfe = new URLSearchParams(params).get('mfe');
-        return ({ data: { ...newConfig.common, ...newConfig[mfe] } });
+        const site = new URLSearchParams(params).get('site');
+        console.log(site);
```
Comment (Nicolas-alt) [inline src/initialize.test.js:304]: This `console.log` is necessary? @dcoa :eyes:

Comment (dcoa) [reply src/initialize.test.js]: Nop, my bad, I fix it, thanks Nico

**Comment by MaferMazu** [comment]
@dcoa, do you have how to test this and the expected result, or is it only a code review? 👀

**Comment by dcoa** [comment]
> @dcoa, do you have how to test this and the expected result, or is it only a code review?

Only code review

**Comment by MaferMazu** [comment]
Looks good to me @dcoa ✨

**Review by MaferMazu — APPROVED** [review]

**Review by JuanDavidBuitrago — APPROVED** [review]
It looks good.

**Review by Nicolas-alt — APPROVED** [review]

**Review by Nicolas-alt — APPROVED** [review]

**Pull request closed without merging**